NineChronicles.Headless.Tests/GraphTypes/StandaloneQueryTest.cs

```diff
@@ -556,6 +558,268 @@ public async Task GetTx()
             Assert.Equal(transaction.Actions.First().PlainValue.Inspection, plainValue);
         }
 
+
+        [Fact]
+        public async Task StateQueryAvatarState()
+        {
+            var userPrivateKey = new PrivateKey();
+            var userAddress = userPrivateKey.PublicKey.ToAddress();
+            var avatarAddress = userAddress.Derive(string.Format(CultureInfo.InvariantCulture,
+                CreateAvatar2.DeriveFormat, 0));
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
+            StandaloneContextFx.BlockChain = service.Swarm.BlockChain;
+
+            var blockChain = StandaloneContextFx.BlockChain;
+            var query = $@"query {{
+                stateQuery {{
+                    avatar(address: ""{avatarAddress}"") {{
+                        address
+                        agentAddress
+                    }}
+                }}
+            }}";
+            var queryResult = await ExecuteQueryAsync(query);
+            Assert.Equal(
+                new Dictionary<string, object>
+                {
+                    ["stateQuery"] =new Dictionary<string, object>
+                    {
+                        ["avatar"] = null,
+                    }
+                },
+                queryResult.Data
+            );
+
+            var action = new SetAvatarState();
+            var tx = StandaloneContextFx.BlockChain.MakeTransaction(
+                userPrivateKey,
+                new PolymorphicAction<ActionBase>[] { action }
+            );
+            await blockChain.MineBlock(userAddress);
+
+            queryResult = await ExecuteQueryAsync(query);
+            Assert.Equal(
+                new Dictionary<string, object>
+                {
+                    ["stateQuery"] =new Dictionary<string, object>
+                    {
+                        ["avatar"] = new Dictionary<string, object>
+                        {
+                            ["address"] = avatarAddress.ToString(),
+                            ["agentAddress"] = userAddress.ToString(),
+                        },
+                    },
+                },
+                queryResult.Data
+            );
+        }
+
+        [Fact]
+        public async Task StateQueryRankingMapState()
+        {
+            var userPrivateKey = new PrivateKey();
+            var userAddress = userPrivateKey.PublicKey.ToAddress();
+            var avatarAddress = userAddress.Derive(string.Format(CultureInfo.InvariantCulture,
+                CreateAvatar2.DeriveFormat, 0));
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
```

Comment:
In my thought, `GraphType` test doesn't need `NineChroniclesNodeService`. how about using `BlockChain<T>` directly?

Reply:
i think using `MakeMineChroniclesNodeService` is more convenient than using `BlockChain<T>`. (for testing config, make genesis, etc)

Reply:
I agree with your opinion but there are some lines not needed for the purpose and it seems able to be resolved next pull request.